### DISCUSSION TITLE
Programação em Lua (baseada em script)

### DISCUSSION THREAD

**Discussion by ghost** [wezlley126/Starfood#12] [Ideas]
https://www.lua.org/portugues.html#:~:text=O%20que%20%C3%A9%20Lua%3F,dados%20e%20descri%C3%A7%C3%A3o%20de%20dados.

**Para quem quiser passar o tempo e rever conceitos básicos de programação com uma linguagem leve, simples e ainda por cima e não menos importante, criada por BRASILEIROS...o que facilita de certa forma, o aprendizado.**

**Comment by ghost** [comment]
Ela tem uma versão do ecplise que foi aquela IDE que a gente usou no primeiro ano para programar em Java e tbm é possível usar o próprio VScode com algumas extensões. Já o Atom, eu não sei se é possível mas creio que sim.

**Comment by WeslleyBorges1267** [comment]
Se tem script vamo botar no site de algum jeito só pq nós quer mesmo.

**Reply by WeslleyBorges1267** [reply]
Deixar o site todo bugado com varias linguagens entrando em conflito.

**Reply by ghost** [reply]
> Se tem script vamo botar no site de algum jeito só pq nós quer mesmo.

Kkkkkk..mds...
Vamos focar só em JS, PHP, HTML, CSS E SQL msm por agr no caso

**Reply by ghost** [reply]
> Deixar o site todo bugado com varias linguagens entrando em conflito.

Sksksks nam
